pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: qr_reader
+description: Implementation of the listener QR scanners that handle output as keyboard.
```

Comment:
Implementation of the QR scanner listener that handles output as keyboard [events].